lib/serviceClient.ts

```diff
@@ -427,3 +425,27 @@ function getOperationArgumentValueFromParameterPath(operationArguments: Operatio
   }
   return value;
 }
+
+interface PropertySearchResult {
+  propertyValue?: any;
+  propertyFound: boolean;
+}
+
+function getPropertyFromParameterPath(parent: { [parameterName: string]: any }, parameterPath: string[]): PropertySearchResult {
+  const result: PropertySearchResult = { propertyFound: false };
+  let i = 0;
+  for (; i < parameterPath.length; ++i) {
+    const parameterPathPart: string = parameterPath[i];
+    // Make sure to check inherited properties too, so don't use hasOwnProperty().
```

Comment:
`in parent` means that the parent contains a property with specifically that name?

Reply:
Yup. That's what I want.